examples/polarion-run-report/test.fmf

```diff
@@ -0,0 +1,15 @@
+summary: Simple test demonstrating Polarion report integration
```

Comment:
Please make this a distinct tmt tree so that it does not appear in our other tests.